src/test/java/bio/terra/service/resourcemanagement/azure/AzureResourceDaoTest.java

```diff
@@ -139,4 +139,32 @@ public void oneApplicationOneBillingProfilesTwoStorageAccounts() throws Exceptio
               equalTo(sa));
         });
   }
+
+  @Test
+  public void markForDeleteStorageAccountMetadata() {
+    var appDeploymentId = applicationDeployments.get(0).getId();
+    assertThat(
+        "Before marking for delete, confirm that 2 storage accounts are returned when marked_for_delete = false",
+        azureResourceDao
+            .retrieveStorageAccountsByApplicationResource(appDeploymentId, false)
+            .size(),
+        equalTo(2));
```

Comment:
There is a built-in matcher for collection size, `hasSize()`.
```suggestion
    assertThat(
        "Before marking for delete, confirm that 2 storage accounts are returned when marked_for_delete = false",
        azureResourceDao
            .retrieveStorageAccountsByApplicationResource(appDeploymentId, false),
        hasSize(2));
```

Reply:
Oh right, thanks!